### PR TITLE
Change behaviour

### PR DESCRIPTION
**Pull request by martinechtner**
- For data coming in stringify the keys so the client side doesn’t have to worry about it. 

- Updated attributes to reflect changes in Maropost.

- Updated README usage.

- Updated Gem version

### CODE REVIEW

**Comment by rubene** [comment]
:book:

**Comment by rubene** [comment]
@jbabe we should add travis?

**Comment by rubene** [comment]
🐘 🐘 🐘

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
...UlPSExLYTdWeTBDUlkyV2NoZ1BaZnJFdnV4RGZQQwpDSjdKdnFPb2hOY09MVnBzeUxSMm5ERUdnZjlNTTB0QkNJMTBtSnJSeE5PYW9FY01ST2syTmMwQnV0M0lTTkp1CmZmNUV1MTlPVHQ0TG8wUGJVaTdCNUgrSlVFZHVLbzMvR0lvVVdnVSttbmtCWGRQZEliVUlYNTBDZ1lFQTZTdlAKUHJIVmVxOU5PK1c3QXBqaTY2QXl0NjkzSThHSHZacm44aGVobld5S1R0OEdQeCtQWUpEblV6UUNGS0Evdm5HOApoWDNydGJpcE9qbVlFYkwzWnpBVG9hNGxGOU9UK3JmbC9jbDJaeC9ZYmIzNUh2SlNpZS90bGFkU1Zid0gzZnBqCldyeUNLSnBYOEFpRklNNDBGSWV5cWtsWG1wcHlQd1M1VnFYc0ZMa0NnWUVBM3dWQ3Bkclg4WjdPSXVVZ0U2d2oKVzdNYlM4QnVDKytQRnBBcVUrSWdOc2FpZTJBam1WNGFSR2laQVlUWGRYa1pUT0E4eXd1K2VwT2tMR2ZnYU0xcwpBSzhobXo4b0UxTlppdzN3ZkxxTWtOZTNNam9IaE5BM3AvSVZ2MWtJWVZ2MWhBQWFGY1NLQlRBYkY5azhpekxFCnl5akd2OHp5dkpBR1JYbjd3SlBsUGFFQ2dZQXdYMkpUUTFsaVpsUERoVmxTcEowTy9PRUlzbUlwRE5FNFNKYXQKNXJFa2s0dnQ2TjdHNGFsQkEwNjZPQ1BtVzRoZDNnS2FQeEZUVmNmbHlHVC81dnprYjFqalJkbDBJL1NqSXR1WAoxdCsxYlNuZE9uZ2dMMjFvS0ZwRm5ieTdCSk10UGZDMnRqemFaNHF1eFZXNUJJNVJaQVZ3eGw1aU9JQlRDb3ZZCnFmdGNJUUtCZ1FEYnZjUEZrMjFuRXRCZ1gxeHUyOVZSTjJtUlQrTFZ4OWVjK3c2Y25QV2Q1NU5HdWlVUnBrQWEKaW0vY040bHNUTlpGR0MvZmpBWWZBdEpVTHViL0JQRTdPY3lwOVFIYjVsRnRibklyRDhHREFtcW1SZWdrNzgxeQpFWmRPMUZCZ2JScFJzWTVJM1JTbEFRRmlrZmlYL2c1aHlmaG9PTDhKYk1obGRpZmE2UU14SUE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo="
```
Comment (jbabe) [inline .travis.yml]: hey @martinechtner we're gonna have to yank this key since this is a public repo

**Pull request merged**